onnxruntime/server/grpc/grpc_app.cc

```diff
@@ -0,0 +1,25 @@
+#include "grpc_app.h"
+#include <grpcpp/health_check_service_interface.h>
+#include <grpcpp/ext/channelz_service_plugin.h>
+#include <grpcpp/ext/proto_server_reflection_plugin.h>
+namespace onnx_grpc = onnxruntime::server::grpc;
+
+namespace onnxruntime {
+namespace server {
+GRPCApp::GRPCApp(const std::shared_ptr<onnxruntime::server::ServerEnvironment>& env, std::string host, const unsigned short port) : prediction_service_implementation_(env) {
```

Comment:
"std::string host, const unsigned short port" could be changed to
"const std::string& host,  unsigned short port"